model_lab_configs/gitignore.md

```diff
@@ -3,3 +3,4 @@ __pycache__
 /history/*/*
 !/history/*/history.config
 !/history/*/olive_config.json
+/infra
```

Comment:
do we need to ignore them? I am thinking that user could modify them to allow custom deployment

Reply:
but I think once deployed, will we deploy again if we have updates?

Reply:
Sure. Let's keep biceps to allow custom deployment, but ignore cloudconversion.config.json which is used for running.